api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -2318,4 +2324,179 @@ public void testGetWorkspacesDoesNotGetsPublishedIfReader() {
 
     assertThat(workspacesController.getWorkspaces().getBody().getItems().size()).isEqualTo(0);
   }
+
+  private void assertNotebookLockingMetadata(
+      Map<String, String> gcsMetadata,
+      NotebookLockingMetadataResponse expectedResponse,
+      WorkspaceACL acl) {
+
+    final String testWorkspaceNamespace = "test-ns";
+    final String testWorkspaceName = "test-ws";
+    final String testNotebook = "test-notebook.ipynb";
+
+    org.pmiops.workbench.firecloud.model.Workspace fcWorkspace =
+        createFcWorkspace(testWorkspaceNamespace, testWorkspaceName, LOGGED_IN_USER_EMAIL);
+    fcWorkspace.setBucketName(BUCKET_NAME);
+    stubGetWorkspace(fcWorkspace, WorkspaceAccessLevel.OWNER);
+    stubFcGetWorkspaceACL(acl);
+
+    final String testNotebookPath = "notebooks/" + testNotebook;
+    doReturn(gcsMetadata).when(cloudStorageService).getMetadata(BUCKET_NAME, testNotebookPath);
+
+    // this is what I want to do, but it's broken somehow (never fails)
```

Comment:
Looks like the paren is misplaced so this doesn't currently assert anything, should probably be:

```
assertThat(...getBody()).isEqualTo(expected);
```

Reply:
thanks, fixed